tensorboard/backend/event_processing/data_ingester.py

```diff
@@ -196,3 +214,19 @@ def _parse_event_files_spec(logdir_spec):
             path = os.path.realpath(os.path.expanduser(path))
         files[path] = run_name
     return files
+
+
+def _cloud_fs_supported() -> bool:
+    """Checks if the cloud filesystems are supported."""
+    for fs in _CLOUD_FILESYSTEMS:
+        try:
+            if fs not in tf.io.gfile.get_registered_schemes():
+                return False
+        except:
+            # `tf.io.gfile.get_registered_schemes` API is not available,
+            # fall back to `tf.io.gfile.exists`.
+            try:
+                tf.io.gfile.exists(fs + "://tmp")
+            except (tf.errors.UnimplementedError, NotImplementedError):
```

Comment:
Hmm, I thought this API should only ever raise tf.errors.UnimplementedError, rather than `NotImplementedError`.  Did you see it raising NotImplementedError in practice?

Reply:
Removed.